apps/home/models.py

```diff
@@ -105,7 +105,9 @@ class HomePage(MetadataPageMixin, Page):
                      'apps_home.SimplePage',
                      'apps_gruenbuch.GruenbuchOverviewPage',
                      'apps_forms.ContactFormPage',
-                     'apps_forms.ParticipationFormPage']
+                     'apps_forms.ParticipationFormPage',
+                     'apps_home.MicrositeOverviewPage',
+                     'apps_home.MicrositeDetailPage']
```

Comment:
do we want the homepage to have microsite detail pages as direct children?

Reply:
I wasnt sure, but as we allow DetailPages as well and the editors will only be able to edit pages that are already there (as they are not allowed to edit homepage), I thought its more flexible this way?